frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditTaskRecommendedItems/NewNameRecommendation.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { NewNameRecommendation } from './NewNameRecommendation';
+import { useBpmnContext } from '../../../../contexts/BpmnContext';
+import { useValidateBpmnTaskId } from '../../../../hooks/useValidateBpmnId';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import userEvent from '@testing-library/user-event';
+import { useStudioRecommendedNextActionContext } from '@studio/components';
+
+jest.mock('../../../../contexts/BpmnContext', () => ({
+  useBpmnContext: jest.fn(),
+}));
+
+jest.mock('../../../../hooks/useValidateBpmnId', () => ({
+  useValidateBpmnTaskId: jest.fn(),
+}));
+
+jest.mock(
+  '@studio/components/src/components/StudioRecommendedNextAction/context/useStudioRecommendedNextActionContext.ts',
+  () => ({
+    useStudioRecommendedNextActionContext: jest.fn(),
+  }),
+);
+
+describe('NewNameRecommendation', () => {
+  const user = userEvent.setup();
+  const setBpmnDetails = jest.fn();
+  const validateBpmnTaskId = jest.fn();
+  const DEFAULT_ID = 'test_id';
+
+  beforeEach(() => {
+    (useBpmnContext as jest.Mock).mockReturnValue({
+      bpmnDetails: { id: DEFAULT_ID, element: {}, metadata: {} },
+      setBpmnDetails: setBpmnDetails,
+      modelerRef: { current: { get: () => ({ updateProperties: jest.fn() }) } },
+    });
+    (useStudioRecommendedNextActionContext as jest.Mock).mockReturnValue({
+      removeAction: jest.fn(),
+      addAction: jest.fn(),
+      shouldDisplayAction: jest.fn(),
+    });
```

Comment:
What about creating a renderNewCommendedItems method which wraps the context? Then I do not think we need this mock at all? See f.ex `renderConfigContent` within `ConfigContent.tsx`.